src/phoenix/server/api/queries.py

```diff
@@ -496,7 +496,10 @@ async def viewer(self, info: Info[Context, None]) -> Optional[User]:
                 )
             ) is None:
                 return None
-        return to_gql_user(user)
+            api_keys = list(
+                await session.scalars(select(models.ApiKey).where(models.ApiKey.user_id == user.id))
+            )
+        return to_gql_user(user, api_keys=api_keys)
```

Comment:
this is graphql over-fetching. It should be a resolver on the User node. That way the keys are only loaded when the query contains it.